java/drivers/driver-hazelcast4plus/src/main/java/com/hazelcast/simulator/tests/map/prunability/ScanByPrunedCompositeKeyBenchmarkBase.java

```diff
@@ -88,9 +94,9 @@ public void prepare() {
 
     @TimeStep
     public void timeStep() throws Exception {
-        final int i = prepareKey();
-        final long l = i;
-        try (SqlResult result = sqlService.execute(query, i, l)) {
+        key = prepareKey();
```

Comment:
you update `key` field, which makes it needlessly complex to understand that it _currently_ does not matter and easier to break it in the future